images/federation-v2.yml

```diff
@@ -9,6 +9,6 @@ from:
   builder:
   - stream: golang
   member: openshift-enterprise-base
-name: openshift/ose-federation-v2
```

Comment:
I think this is wrong though. Shouldn't it be `-operator`?

Reply:
Paul is working on this, would drop it from this change.